internal/providers/terraform/google/storage_bucket.go

```diff
@@ -169,7 +174,7 @@ func dataRetrieval(d *schema.ResourceData, u *schema.UsageData) *schema.CostComp
 	resourceGroup := storageClassResourceGroupMap[storageClass]
 	// Skipping standard, regional and multi-regional since they are free
 	if resourceGroup == "" {
-		return &schema.CostComponent{IgnoreIfMissingPrice: true}
```

Comment:
@sinabakh FYI `IgnoreIfMissingPrice` means: look it up in the pricing API, if it's not there then don't add it. We use it for EBS Optimized instances since we don't know if they should have that cost component without looking it up. This doesn't seem like that use-case so I changed the code to not add the cost component if it's nil.

Reply:
@alikhajeh1 I used nil first, but it raised a pointer exception. I thought it's a design choice. I think it's shipped with this bug, I will test it and report it as a bug.

Reply:
@alikhajeh1 It was my mistake, it must have been some other problem. Thanks for fixing it.